packages/riipen-ui/src/components/MenuList.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+import clsx from "clsx";
+
+import MenuItem from "./MenuItem";
+
+import List from "./List";
+
+class MenuList extends React.Component {
+  static propTypes = {
+    /**
+     * The content of the component.
+     */
+    children: PropTypes.arrayOf(
+      PropTypes.shape({
+        type: PropTypes.oneOf([MenuItem])
+      })
+    ),
+
+    /**
+     * Wether to select the first element on list open
+     */
+    autoFocus: PropTypes.bool,
+
+    /**
+     * The selected index of the list
+     */
+    selectedIndex: PropTypes.number,
+
+    /**
+     * The function callback when an item is selected
+     */
+    selectChange: PropTypes.func,
```

Comment:
Just sticking with conventions we have, can we rename this to either `onChange` or `onSelect`?